src/main/java/io/percy/selenium/Percy.java

```diff
@@ -164,6 +170,56 @@ public void snapshot(String name, Map<String, Object> options) {
         postSnapshot(domSnapshot, name, driver.getCurrentUrl(), options);
     }
 
+    public void screenshot(String name) {
```

Comment:
we need to give user an option to pass options, refer to snapshot command for the same.